source/database/mssql/scripts/dbscripts/PSP_PIMS_S50_00/Build/121_DML_PIMS_ACQ_CHKLST_ITEM_TYPE.sql

```diff
@@ -11,38 +11,44 @@ GO
-  (N'SCTN6XPRP', N'CLAIMRELSS6', N'Release of Claims',                                        33, getutcdate(), ''),
-  (N'ACQCOMPAC', N'PIMSUPDTFIL', N'File data entered into PIMS',                              34, getutcdate(), 'Data is complete in this system');
+  (N'FILEINIT',  N'PREACQTTL',   N'Pre-acquisition title',                                      1, getutcdate(), NULL),
+  (N'FILEINIT',  N'BCASSESS',    N'BC Assessment',                                              2, getutcdate(), NULL),
+  (N'FILEINIT',  N'COMPSRCH',    N'Company search',                                             3, getutcdate(), NULL),
+  (N'FILEINIT',  N'LANDTTL',     N'Land Title documents or plans',                              4, getutcdate(), NULL),
+  (N'FILEINIT',  N'SGNPAPLAN',   N'Property Acquisition (PA) Plan',                             5, getutcdate(), 'A signed copy of the PA Plan is required'),
+  (N'FILEMGMT',  N'PHOTOS',      N'Photos',                                                     6, getutcdate(), NULL),
+  (N'FILEMGMT',  N'APPRAISE',    N'Appraisals and reviews',                                     7, getutcdate(), NULL),
+  (N'FILEMGMT',  N'OTHERRPTS',   N'Other reports',                                              8, getutcdate(), 'Final copies of professional reports (e.g. Environmental reports)'),
+  (N'FILEMGMT',  N'RECNEGOT',    N'Record of negotiation',                                      9, getutcdate(), NULL),
+  (N'FILEMGMT',  N'CORRESCPY',   N'Copies of correspondence',                                  10, getutcdate(), NULL),
+  (N'FILEMGMT',  N'SAASIGNED',   N'Spending Authority Approval(s) - SAA',                      11, getutcdate(), 'The signed SAA is required'),
+  (N'FILEMGMT',  N'FINALAGRM',   N'Final agreements with owner(s)',                            12, getutcdate(), 'H179 P/T/A, TLCA, LTC, etc.'),
+  (N'FILEMGMT',  N'H120COMPREQ', N'H120(s) processed for compensation',                        13, getutcdate(), NULL),
+  (N'FILEMGMT',  N'LEGALDOCS',   N'Instructions or documents sent to lawyer',                  14, getutcdate(), NULL),
+  (N'FILEMGMT',  N'OWNERCOMP',   N'Copy of owners'' compensation cheque(s)',                   15, getutcdate(), NULL),
+  (N'FILEMGMT',  N'ENTRYCOND',   N'Conditions of Entry (H0443)',                               16, getutcdate(), NULL),
+  (N'FILEMGMT',  N'CONVEYCLOS',  N'Conveyance closing documents',                              17, getutcdate(), NULL),
+  (N'FILEMGMT',  N'REGSTRDPLAN', N'Registered Plan',                                           18, getutcdate(), 'Letter size (8.5" x 11")'),
+  (N'FILEMGMT',  N'NEWTTLACOMP', N'Copy of new title',                                         19, getutcdate(), 'Copy of new title after completion'),
+  (N'FILEMGMT',  N'BCTFANOTIF',  N'BCTFA notified',                                            20, getutcdate(), 'Applicable if this a total take with a surplus'),
```

Comment:
The alter Up script looks updated according to @asanchezr comment. Has the Build script been updated too?